packages/superset-ui-demo/storybook/stories/plugins/legacy-plugin-chart-calendar/Stories.tsx

```diff
@@ -1,15 +1,14 @@
-/* eslint-disable sort-keys, no-magic-numbers */
 import React from 'react';
 import { SuperChart } from '@superset-ui/chart';
-import CalendarChartPlugin from '../../../../../../plugins/legacy-plugin-chart-calendar';
+import CalendarChartPlugin from '../../../../../../plugins/legacy-plugin-chart-calendar/lib';
```

Comment:
Can change this to `import CalendarChartPlugin from '@superset-ui/legacy-plugin-chart-calendar';`

Symlink from [yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/) should take care of importing from source.